### PR TITLE
Enhancement: Add `SECURITY.md`

### PR DESCRIPTION
**Pull request by localheinz**
### What is the reason for this PR?

- [x] add a `SECURITY.md`

Also see #740 and #744.

💁‍♂️ While the primary purpose of a `SECURITY.md` is to inform users of `fakerphp/faker` about our security policy, this file could also serve well as a starting point for a discussion which versions of `fakerphp/faker` we currently support.

I may have missed it, but I have not found such a document in this repository.

### Author's checklist

- [x] Follow the [Contribution Guide](https://github.com/FakerPHP/Faker/blob/main/.github/CONTRIBUTING.md)

### Review checklist

- [x] All checks have passed
- [ ] Changes are added to the `CHANGELOG.md`
- [ ] Changes are approved by maintainer

### CODE REVIEW

File: .github/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## Unsupported Versions
+
+The following versions of `fakerphp/faker` have reached their end of life:
+
+- `<1.23.0`
+
+## Reporting a Vulnerability
+
+If you believe that you have found a security vulnerability, please send an email to ?. Ensure to include all details required to understand the severity of the issue.
```
Comment (localheinz) [inline .github/SECURITY.md:17]: Who would like to receive emails about security issues related to `fakerphp/faker`?

Comment (pimjansen) [reply .github/SECURITY.md:17]: I can check if we can register a fqdn for this? Setting up a mailbox feels better to link it directly to the lib. What do you think?

Comment (GrahamCampbell) [reply .github/SECURITY.md:17]: you can send em to me if you want. you could also just list multiple emails to send to - i've done that before with projects. hello at gjcampbell.co.uk

Comment (localheinz) [reply .github/SECURITY.md:17]: I prefer an email address that is not tied to a single person.

Comment (GrahamCampbell) [reply .github/SECURITY.md:17]: probably we should list multiple email addresses, then

Comment (bram-pkg) [reply .github/SECURITY.md:17]: That's easily doable with Cloudflare email routing, that's probably fine?

Comment (bram-pkg) [reply .github/SECURITY.md:17]: I've send a message in slack to discuss a domain name registration, I can register it and add you all to the cloudflare dashboard for it.

Comment (localheinz) [reply .github/SECURITY.md:17]: @bram-pkg 

Maybe a single email address will suffice, perhaps `security@fakerphp.org`?

**Review by bram-pkg — APPROVED** [review]

**Comment by localheinz** [comment]
Thank you, @bram-pkg, @GrahamCampbell, and @pimjansen!

**Pull request merged**